components/mpas-ocean/src/shared/mpas_ocn_gm.F

```diff
@@ -663,6 +681,38 @@ subroutine ocn_GM_compute_Bolus_velocity(statePool, &
          endif
 
          nEdges = nEdgesArray(3)
+         if(config_gm_enable_horizontal_resolution_function) then
```

Comment:
@mark-petersen: if we decide to remove the config option above and just add a 'rossbyRadius' option to `config_eddying_resolution_taper`, then the statement above would become:
`if (config_eddying_resolution_taper == 'rossbyRadius') then`

But before going down this line, I'd make sure the change is blessed by @vanroekel.